tests/integration-tests/tests/common/utils.py

```diff
@@ -272,13 +272,36 @@ def reboot_head_node(cluster, remote_command_executor=None):
     logging.info(f"Rebooted head node for cluster: {cluster.name}")
 
 
+def reboot_login_node(cluster, remote_command_executor=None):
+    logging.info(f"Rebooting login node for cluster: {cluster.name}")
+    if not remote_command_executor:
+        remote_command_executor = RemoteCommandExecutor(cluster, use_login_node=True)
+    command = "sudo reboot"
+    result = remote_command_executor.run_remote_command(command, raise_on_error=False)
+    logging.info(f"result.failed={result.failed}")
+    logging.info(f"result.stdout={result.stdout}")
+    wait_login_node_running(cluster)
```

Comment:
I think we should not only waiut for the instance to be running, but also for the instance to be in status ok. See https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/ec2/waiter/InstanceStatusOk.html

Reply:
Added